astro/src/content/blog/announcing-fusionauth-1-56.mdx

```diff
@@ -0,0 +1,76 @@
+---
+publish_date: 2025-03-04
+title: Announcing FusionAuth Version 1.56 - The SSO Snake
+description: FusionAuth v1.55.1 "The Holiday Hippo" brings major updates including SAML encryption support, enhanced API key security with via expiration, configurable lambda timeouts, and more. 
+authors: Brad McCarty
+image: /img/blogs/release-1-56/fusionauth-1-56.png
+categories: Product
+tags: SSO, Custom Domains, API Key Security
+excerpt_separator: "{/* more */}"
+---
+import Aside from '/src/components/Aside.astro';
+
+We recently released FusionAuth version 1.56.0. This version focuses on API security, and it introduces some new flexibility in how you can use SSO sessions within FusionAuth. We also have custom domains for every hosting plan.
+
+{/* more */}
+
+Our fearless Senior Director of Developer Relations ([welcome to the team, Kim](https://www.linkedin.com/posts/fusionauth_this-week-we-are-super-excited-to-welcome-activity-7301000700145131520-FkVA?utm_source=share&utm_medium=member_desktop&rcm=ACoAAAGscwkBMMEEWM3JSitpba873aUwI_0jz6Q)!) has dubbed this release “The SSO Snek.” Our slithering friend is here to sneak past non existent SSO sessions and ensure a seamless experience for your users.
+
+![The SSO Snek is a sneaky little guy](/img/blogs/release-1-56/sso-snake.jpg)
+
+And now, on to the good stuff!
+
+## API Key Security
+
+In version 1.56.0, we’ve improved API key security by introducing hashed storage. This builds on our [1.55.1 release](/blog/announcing-fusionauth-1-55) that added expiration times. With 1.56.0, newly-created API keys are now optionally secured before being stored, reducing risks in case of a database compromise. We want to stress that the default behavior is not changing, so your existing keys will continue to work. If you want your keys secured, you will need to create new ones.
```

Comment:
```suggestion
In version 1.56.0, we’ve improved API key security by introducing secured storage. This builds on our [1.55.1 release](/blog/announcing-fusionauth-1-55) that added expiration times. With 1.56.0, newly-created API keys are now optionally secured before being stored, reducing risks in case of a database compromise. We want to stress that the default behavior is not changing, so your existing keys will continue to work. If you want your keys secured, you will need to create new ones.
```